packages/utils/classes.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Generates a string of unique CSS classes from the provided list,
+ * filtering out non-string values and empty strings.
+ *
+ * Ensures that the output contains only valid, non-empty class names,
+ * and removes any duplicates.
+ *
+ * @param {...string[]} classList - A variable number of CSS class names.
+ *
+ * @return {string} A string of unique, valid CSS classes, separated by spaces.
+ *
+ * @example classes("bg-red-500", "text-white");
+ * >>> "bg-red-500 text-white"
+ *
+ * @example classes("bg-red-500", "bg-red-500");
```

Comment:
here is another valid example/testcase
```
 * @example classes("bg-red-500 w-4", "bg-red-500 px-3");
 * >>> "bg-red-500 w-4 px-3"
```

Reply:
Added in: https://github.com/podcodar/webapp/pull/175/commits/e24fcc25615c1dd8ffa91cf5fdc33df03d947ad9